xtask/Cargo.toml

```diff
@@ -0,0 +1,29 @@
+[package]
+name = "xtask"
+version = "0.1.0"
+authors.workspace = true
+homepage.workspace = true
+repository.workspace = true
+edition.workspace = true
+license.workspace = true
+
+[dependencies]
+
+clap = { version = "4.4.8" , features = ["derive"] }
+anyhow = { version = "1.0.75" }
+
+# benchmarking
+hdrhistogram = "7.5.4"
+fxhash = "0.2.1"
+rand = "0.8.5"
+rand_pcg = "0.3.1"
+sha2 = { version = "0.10.6" }
```

Comment:
why sha2, out of curiousity?

Reply:
No real reason, it is the one used in Sov-DB, so I stick with it for NOMT to have a comparison that leverages mostly tree things. In a future feature, the hashing algorithm used could become a parameter